connutils.go

```diff
@@ -448,6 +490,54 @@ func parseConnectDSNAndArgs(
 		waitUntilAvailable = 30 * time.Second
 	}
 
+	roots, err := getCertPool()
```

Comment:
We should probably not include system certs when the user provides their own certificate right?

Reply:
I think so.

Reply:
changed